lib/ansible/modules/cloud/lxd/lxd_container.py

```diff
@@ -102,6 +102,12 @@
             starting or restarting.
         required: false
         default: false
+    wait_for_ipv4_addresses_ignore_devices:
+        description:
+          - This will add additional devices to the ignore list which
+            currently only ignores the loopback device "lo".
+        required: false
+        version_added: 2.4
```

Comment:
By looking [ansible/yum.py at e9e661ebbbf59c1ab397ab386517242a37e9f857 · ansible/ansible](https://github.com/ansible/ansible/blob/e9e661ebbbf59c1ab397ab386517242a37e9f857/lib/ansible/modules/packaging/os/yum.py#L49) and [ansible/apt.py at e9e661ebbbf59c1ab397ab386517242a37e9f857 · ansible/ansible](https://github.com/ansible/ansible/blob/e9e661ebbbf59c1ab397ab386517242a37e9f857/lib/ansible/modules/packaging/os/apt.py#L92), the value for `version_added` should be a string instead of a float.